scripts/dotnet.py

```diff
@@ -74,7 +75,7 @@ def csproj_file(self) -> str:
     @property
     def bin_path(self) -> str:
```

Comment:
Is this always the case when running the arcade/build or dotnet commands?

Reply:
Yes

Reply:
Then references to `path.join(BENCHMARKS_CSPROJ.working_directory, 'obj')` should be updated.

Reply:
Ok thanks

Reply:
There's only the one reference to the obj directory, and it's in the delete binary folders code.